packages/retail-ui-extensions/src/components/FormattedTextField/FormattedTextField.ts

```diff
@@ -0,0 +1,34 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {AutoCapitalizationType} from '../shared';
+
+export type InputType =
+  | 'text'
+  | 'number'
+  | 'percent'
+  | 'boundlessPercent'
+  | 'currency'
+  | 'giftcard'
+  | 'expiry'
+  | 'date'
+  | 'zip'
+  | 'password'
+  | 'email'
+  | 'url'
+  | 'phone';
+
+export interface FormattedTextFieldProps {
+  inputType?: InputType;
+  title?: string;
```

Comment:
I do see a lot of attributes here that are shared with `TextField.ts`. We could probably stick them into an interface called `BaseTextFieldProps`, and `FormattedTextFieldProps` and the other one can extend the base props. We could put the base props in the shared directory of components